thunder/executors/torch_compile.py

```diff
@@ -63,36 +68,34 @@ def make_compiled(
     from thunder import trace
     from thunder.core.transforms import eval_trace
     from thunder.executors.torchex import no_autocast
+    from thunder.executors.pythonex import ex as pythonex
+    from thunder.core.codeutils import SigInfo
 
     # Here we construct a trace that will be used to compile the function
+    # TODO: maybe we should have a utility that does this properly
     region_trace = TraceCtx(None)
-    region_trace.bound_symbols = list(bsyms)
     region_trace.args = sorted_unique_inputs
     region_trace.kwargs = {}
+    with tracectx(region_trace):
+        for a in sorted_unique_inputs:
+            prims.unpack_trivial(a, name=a.name)
+
+    region_trace.bound_symbols += list(bsyms)
     region_trace.bound_symbols.append(prims.python_return.bind(sorted_unique_outputs, output=None))
+    # for a in region_trace.args:
+    #    region_trace.add_name(a.name)
+    for bsym in region_trace.bound_symbols:
+        for o in bsym.flat_outs:
+            if o is not None:  # TODO: investigate
```

Comment:
Would you elaborate on this TODO -- is the question about what's happening with a bsym with no outputs?

Reply:
I think the todo can be dropped. I was surprised that we get None outputs here, but it is OK.